modules/data-loading/pages/kafka-loader/index.adoc

```diff
@@ -237,6 +237,9 @@ RUN LOADING JOB [-noprint] [-dryrun] [-n [i],j] jobname
 
 To learn about each option and parameter of the `RUN LOADING JOB` command, see xref:gsql-ref:ddl-and-loading:running-a-loading-job.adoc#_options[Loading job options].
 
+The xref:reference:configuration-parameters.adoc#_kafkaloader[Kafka Loader parameter] `KafkaLoader.ReplicaNumber` controls the number of Kafka Loader replicas per node.
+The default is `1`, which means only one loading job can be run at a time. To run multiple loading jobs at the same time, increase this parameter. 
```

Comment:
Since this is number of replicas per node, does it mean if I have a 4 node cluster, I would be able to run 4 loading jobs at a time? Please clarify.

Reply:
This is more like a question to GLE team. I'll paste this to our stream chat

Reply:
<img width="364" alt="image" src="https://user-images.githubusercontent.com/99212064/170577949-7246e08f-f76a-49b9-a6cd-d6037cd780d1.png">
Paste Chengjie's answer here.

Reply:
the KafkaLoader.ReplicaNumber is independent of cluster size, that is to say you may setup this number > 1 on single node TG platform as well. @ryantigergraph @jinhui-tigergraph @chengjie-qin 

On theory, you may set this number to be any value (nothing to do with the cluster size), we have no limit on KafkaLoader.ReplicaNumber as long as Linux CPU/MEM resource is enough